licenses/update_license.py

```diff
@@ -39,10 +39,10 @@ def update_license_in_file(
     start_line_start_with: str,
     end_line_start_with: str,
 ) -> bool:
-    with open(file_path, 'r') as f:
+    with open(file_path, 'r', encoding='utf-8') as f:
```

Comment:
Can you explain why did you add `encoding='utf-8'` in a comment?

Reply:
Because the error I encountered was `UnicodeDecodeError: 'gbk' codec can't decode byte 0x9d in position 145: illegal multibyte sequence`. Maybe you don't remember that I asked you this question in February lol.

Reply:
@FUYICC Yeah. I mean we can add a comment here, since this PR is not related to license nor metions it. It will confuse future developers why we modify `update_license.py` in this PR. Like:
```suggestion
    with open(file_path, 'r', encoding='utf-8') as f: # for windows compatibility
```